lib/account/account.routes.js

```diff
@@ -5,26 +5,40 @@ module.exports = {
     return [
       {
         method: 'GET',
-        path: '/accounts',
-        handler: controllers.getAccount,
+        path: '/currentAccount',
```

Comment:
I think I'd like to keep the api path starting with /accounts, so maybe /accounts/self or accounts/current? This will probably break the front-end though since I believe it is using /accounts for the current account. A better option might be to change getAccounts to be /accounts/all